cmd/debos/debos.go

```diff
@@ -88,6 +88,10 @@ func main() {
 		"no_proxy",
 	}
 
+	if os.Getuid() == 0 {
+		log.Printf("Warning, you should not execute debos as root!")
```

Comment:
Maybe; but then a lot of the debos runs in CI run as root as well (though obviously quite containerized).. So it's not that black and white

Reply:
or maybe to put it better; in CI things often run as uid 0 even it not being the "systems" root